### ISSUE TITLE
[A11Y] Add HTML tag "aria-hidden"

### ISSUE THREAD

**Issue by Mitchnsun** [react-component/collapse#137]
For accessibility, collapse element should have an `aria-hidden` tag on the `div.ReactCollapse--collapse`

Cases: aria-hidden=
- `true` collapse is closed
- `false` collapse is opened

**Comment by afc163** [comment]
PR is welcome~
